[INFO   ] Logger: Record log in /storage/emulated/0/learn kivy/22-lines(circle and Ellipse and regatangle/.kivy/logs/kivy_22-12-21_7.txt
[INFO   ] Kivy: v2.0.0
[INFO   ] Kivy: Installed at "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/__init__.py"
[INFO   ] Python: v3.9.7 (default, Oct  6 2021, 01:34:26) 
[GCC 11.1.0]
[INFO   ] Python: Interpreter at "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/bin/python3"
[INFO   ] Factory: 186 symbols loaded
[INFO   ] Image: Providers: img_tex, img_dds, img_sdl2, img_pil (img_ffpyplayer ignored)
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "/storage/emulated/0/learn kivy/22-lines(circle and Ellipse and regatangle/main.py", line 10, in <module>
[WARNING] stderr:     Main().run()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/app.py", line 949, in run
[WARNING] stderr:     self._run_prepare()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/app.py", line 918, in _run_prepare
[WARNING] stderr:     self.load_kv(filename=self.kv_file)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/app.py", line 691, in load_kv
[WARNING] stderr:     root = Builder.load_file(rfilename)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/lang/builder.py", line 306, in load_file
[WARNING] stderr:     return self.load_string(data, **kwargs)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/lang/builder.py", line 373, in load_string
[WARNING] stderr:     parser = Parser(content=string, filename=fn)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/lang/parser.py", line 402, in __init__
[WARNING] stderr:     self.parse(content)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/lang/parser.py", line 511, in parse
[WARNING] stderr:     objects, remaining_lines = self.parse_level(0, lines)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/lang/parser.py", line 657, in parse_level
[WARNING] stderr:     _objects, _lines = self.parse_level(
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/lang/parser.py", line 566, in parse_level
[WARNING] stderr:     raise ParserException(self, ln,
[WARNING] stderr: kivy.lang.parser.ParserException: Parser: File "/storage/emulated/0/learn kivy/22-lines(circle and Ellipse and regatangle/main.kv", line 7:
[WARNING] stderr: ...
[WARNING] stderr:       5:            rgba:1,0,0,1
[WARNING] stderr:       6:        Line:
[WARNING] stderr: >>    7:             circle:(200,200)
[WARNING] stderr:       8:             
[WARNING] stderr: ...
[WARNING] stderr: Invalid indentation, must be a multiple of 4 spaces
